refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/IsEmptyTest.java

```diff
@@ -0,0 +1,443 @@
+            "  int[][] negative4() {",
+            "    return new int[][] {{0}};",
+            "  }",
+            "",
+            "  Random negative5() {",
+            "    return new Random();",
+            "  }",
+            "",
+            "  List<Integer> negative6() {",
+            "    return new ArrayList<>(ImmutableList.of(1));",
+            "  }",
+            "",
+            "  Map<Integer, Integer> negative7() {",
+            "    return new HashMap<>(ImmutableMap.of(1, 2));",
+            "  }",
+            "",
+            "  Set<Integer> negative8() {",
+            "    return new HashSet<>(ImmutableList.of(1));",
+            "  }",
+            "",
+            "  Map<Integer, Integer> negative9() {",
+            "    return new LinkedHashMap<>(ImmutableMap.of(1, 2));",
+            "  }",
+            "",
+            "  Set<Integer> negative10() {",
+            "    return new LinkedHashSet<>(ImmutableList.of(1));",
+            "  }",
+            "",
+            "  List<Integer> negative11() {",
+            "    return new LinkedList<>(ImmutableList.of(1));",
+            "  }",
+            "",
+            "  Map<Integer, Integer> negative12() {",
+            "    return new HashMap<>(ImmutableMap.of(1, 2));",
+            "  }",
+            "",
+            "  Set<Integer> negative13() {",
+            "    return new HashSet<>(ImmutableList.of(1));",
+            "  }",
+            "",
+            "  List<Integer> negative14() {",
+            "    return new Vector<>(ImmutableList.of(1));",
+            "  }",
+            "",
+            "  ImmutableList<Integer> negative15() {",
+            "    return ImmutableList.of(1);",
+            "  }",
+            "",
+            "  ImmutableSet<Integer> negative16() {",
+            "    return ImmutableSet.of(1);",
+            "  }",
+            "",
+            "  ImmutableMap<Integer, Integer> negative17() {",
+            "    return ImmutableMap.of(1, 2);",
+            "  }",
+            "",
+            "  ImmutableSetMultimap<Integer, Integer> negative18() {",
+            "    return ImmutableSetMultimap.of(1, 2);",
+            "  }",
+            "",
+            "  List<Integer> negative19() {",
+            "    return List.of(1);",
+            "  }",
+            "",
+            "  Map<Integer, Integer> negative20() {",
+            "    return Map.of(1, 2);",
+            "  }",
+            "",
+            "  Set<Integer> negative21() {",
+            "    return Set.of(1);",
+            "  }",
+            "",
+            "  Stream<Integer> negative22() {",
+            "    return Stream.of(1);",
+            "  }",
+            "",
+            "  int[] positive1() {",
+            "    // BUG: Diagnostic contains:",
+            "    return new int[0];",
+            "  }",
+            "",
+            "  int[][] positive2() {",
+            "    // BUG: Diagnostic contains:",
+            "    return new int[0][1];",
```

Comment:
And the `[1][0]` would maybe be interesting as that will fall through the first check of `.get(0) == 0`.

Reply:
I'll have a closer look at your suggestion, but to the general point of method naming: that was quite painful for this PR indeed. I tried to remain consistent with other tests, but it did make me wonder: perhaps we should drop all those number suffixes in favour of something more descriptive.

(W.r.t. this PR: at some point I renumbered using a Vim macro ;).)

Reply:
W.r.t. the suggestion itself: `return new int[1][0]` is used above, in method `negative2` :)